fixed #8 - Disabled body scroll when dialog open, then it gets opened

destroycompiler opened the pull request:
fixed #8 : Scroll Disabled when dialog open
---
- Created 2 JavaScript functions, `disableBodyScroll()` and `enableBodyScroll()`.
- Scroll gets disabled when the dialog opens.
- Scroll gets enabled when the dialog closes.

- I have achieved this by utilising the `overflow:hidden` property of the `body` element.
---
## Following are the screenshots after solving the issue

> Scroll Disabled
![Screenshot_20230206_140917](https://user-images.githubusercontent.com/11803841/216925615-1dfcdf6b-6611-41e8-8b35-aaa4a4eccf07.png)

> Scroll Enabled
![Screenshot_20230206_141004](https://user-images.githubusercontent.com/11803841/216925853-5d80fc84-456e-4491-aef9-c74a6e04fb94.png)

chakri68 commented:
Great Work!

Right now, it's jumping to the top when you're toggling the style `overflow`. If you make it not jump up, I can give you bonus points (7).

destroycompiler commented:
I have fixed the bug in my repo's latest commit.
---
> Picture of the fix
---
![Screenshot_20230206_170008](https://user-images.githubusercontent.com/11803841/216960623-c319c7f4-de92-40e5-8bfd-a77833beb386.png)
---

### Changes

- Changed the `position:relative;` attribute of the dialog box to `positon:fixed` so that it doesn't position itself relative to the parent. Now it appears centered without jumping to the top.

chakri68 commented:
Did you commit and push the files?

chakri68 commented:
Fun fact: `dialog` element has `position: fixed` by default ⭐

<p align="center">
<img width="350" alt="image" src="https://user-images.githubusercontent.com/99131274/216963835-cf863216-b406-476c-b763-75d4cf25443e.png">
</p>

Great work though 👍
